packages/medusa/src/types/modules.ts

```diff
@@ -0,0 +1,4 @@
+export type ModulesResponse = {
```

Comment:
question: Can't it be `modules` it look really link to the response here but I am not sure it should. wdyt?